Assets/MRTK/Tests/PlayModeTests/ProfileTests.cs

```diff
@@ -98,6 +98,38 @@ private List<IMixedRealityPointer> GetAllPointers()
             }
             return allPointers;
         }
+
+        /// <summary>
+        /// Tests that the camera system initializes with the floor height specified in the profile.
+        /// </summary>
+        /// <returns>enumerator for Unity</returns>
+        [UnityTest]
+        public IEnumerator TestProfileFloorHeight()
+        {
+            float floorHeight = 3.0f;
+
+            var hl1Profile = ScriptableObjectExtensions.GetAllInstances<MixedRealityToolkitConfigurationProfile>()
+            .FirstOrDefault(x => x.name.Equals(HoloLens1ProfileName));
+
+            // keep the old floor height to reset it later
+            float oldFloorHeight = hl1Profile.ExperienceSettingsProfile.FloorHeight;
+            hl1Profile.ExperienceSettingsProfile.TargetExperienceScale = ExperienceScale.Room;
+            hl1Profile.ExperienceSettingsProfile.FloorHeight = floorHeight;
+            TestUtilities.InitializeMixedRealityToolkit(hl1Profile);
```

Comment:
If we're having to rewrite a bunch of tests like this, it's starting to feel like we aren't sufficiently leaving the default state of the world in a backwards compatible state. If our tests need changes, this tells me developers will need to make changes to keep the way their app currently behaves. Is that the case?

Reply:
This is explicitly to properly test teleport with the new floor height feature, and these adjustments are just to make sure that our changes to the default profile aren't saved after the test finishes (the default profile should match the current dev experience, floor height 0, room scale VR)